_posts/2021-02-24-#21224.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: pr
+date: 2021-02-24
+title: "Halt processing of unrequested transactions"
+pr: 21224
+authors: [ariard]
+components: ["p2p"]
+host: ellemouton
+status: upcoming
+commit:
+---
+
+## Notes
+
+- Currently it is possible for peers to send each other `tx` messages even if the
+  transaction with that txid/wtxid has not been requested.
+
+- This PR aims to change that by requiring a transaction to first be requested
+  by a node before it is relayed to that node from one of its peers.
+
+- See the PR’s author's motivation behind the PR along with some of the
+  concerns that this change brings about:
+  https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-February/018391.html
+
+## Questions
+
+1. Did you review the PR? Concept ACK, approach ACK, tested ACK, or NACK?
+
+2. What is the TxRequestTracker class used for?
```

Comment:
Maybe add a link to `src/txrequest.h`? Use the latest master commit for the url.